autogpt_platform/docker-compose.platform.yml

```diff
@@ -134,6 +135,8 @@ services:
       - REDIS_PASSWORD=password
       - ENABLE_AUTH=true
       - PYRO_HOST=0.0.0.0
+      - BACKEND_CORS_ALLOW_ORIGINS=["http://localhost:3000"]
```

Comment:
Why the .env.example and the other places use comma separates list but here we use json

Reply:
Because we have 2 systems -  marketplace and backend. Backend has config being read via pydantic, marketplace does not have that setup.

This PR is focused on fixing COR's and Firefox issues. Rather than dealing with technical debt